src/AppInstallerCLICore/Workflows/FontFlow.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "FontFlow.h"
+#include "TableOutput.h"
+#include <winget/Fonts.h>
+
+namespace AppInstaller::CLI::Workflow
+{
+    using namespace AppInstaller::CLI::Execution;
+
+    namespace
+    {
+        struct InstalledFontFamiliesTableLine
+        {
+            InstalledFontFamiliesTableLine(Utility::LocIndString familyName, int faceCount)
+                : FamilyName(familyName), FaceCount(faceCount) {}
+
+            Utility::LocIndString FamilyName;
+            int FaceCount;
+        };
+
+        struct InstalledFontFacesTableLine
+        {
+            InstalledFontFacesTableLine(Utility::LocIndString faceName, Utility::LocIndString familyName, std::filesystem::path filePath)
+                : FaceName(faceName), FamilyName(familyName), FilePath(filePath) {}
+
+            Utility::LocIndString FaceName;
+            Utility::LocIndString FamilyName;
+            std::filesystem::path FilePath;
+        };
+
+        void OutputInstalledFontFamiliesTable(Execution::Context& context, const std::vector<InstalledFontFamiliesTableLine>& lines)
+        {
+            Execution::TableOutput<2> table(context.Reporter, { Resource::String::FontFamily, Resource::String::FontFaceCount });
+
+            for (const auto& line : lines)
+            {
+                table.OutputLine({ line.FamilyName, std::to_string(line.FaceCount) });
+            }
+
+            table.Complete();
+        }
+
+        void OutputInstalledFontFacesTable(Execution::Context& context, const std::vector<InstalledFontFacesTableLine>& lines)
+        {
+            Execution::TableOutput<3> table(context.Reporter, { Resource::String::FontFace, Resource::String::FontFamily, Resource::String::FontFilePaths });
+
+            for (const auto& line : lines)
+            {
+                table.OutputLine({ line.FaceName, line.FamilyName, line.FilePath.u8string() });
+            }
+
+            table.Complete();
+        }
+    }
+
+    void ReportInstalledFontsResult(Execution::Context& context)
+    {
+        if (context.Args.Contains(Args::Type::FamilyName))
```

Comment:
Will there be any provision for searching by Face Name? Not sure if it provides value or not

Reply:
Yup, will come out in a later PR when I implement a searchable font source.